eslint/babel-eslint-config-internal/package.json

```diff
@@ -19,6 +19,6 @@
   },
   "peerDependencies": {
     "@babel/eslint-parser": "^7.10.4",
-    "eslint-plugin-flowtype": "^3.0.0"
+    "eslint-plugin-flowtype": "^4.6.0"
```

Comment:
`eslint-plugin-flowtypw@4.6.0` is installed in root package.json so we provided `4.6.0` for `^3.0.0`, since `@babel/eslint-config-internal` is private, we bump the peerDependencies to what we used in root package.json.